schema/transportation/segment.yaml

```diff
@@ -258,17 +257,18 @@ properties:
                        - gatedKey
                      default:
                        enum: [legal]
-                   direction:
-                     description: direction of travel on the source that pertains to the relation
-                     "$ref": "#/$defs/propertyDefinitions/direction"
-                   via:
+                   sequence:
                      description: Sequence of ordered segements that needs to be traversed in order for the relation to be effective
                      type: array
                      items:
-                       description: GERS ID of the connected segments
+                       description: GERS ID of the connected segments and the relevent connector
                        "$ref": "#/$defs/propertyDefinitions/relationTarget"
                      minItems: 1
                      uniqueItems: true
+                   finalHeading:
+                     description: direction of travel on the target that pertains to the relation
+                     "$ref": "#/$defs/propertyDefinitions/direction"
```

Comment:
Shouldn't this type be renamed from `direction` to `heading`?

Also, `direction` is [currently defined to contain](https://github.com/OvertureMaps/schema/blob/3a5e5b62e43a19ae1d24b9cddb3af149003295f7/schema/transportation/segment.yaml#L96) `forward`, `backward`, and `both`. Should the `heading` type just be `forward` and `backward` and should we drop the `both`? Or do we need `both`? Maybe we do...?

Reply:
We do not have a heading enum, we have heading container, which hosts the direction enum. finalHeading is of type direction because otherwise it becomes heading: <direction>